guides/source/ruby_on_rails_guides_guidelines.md

```diff
@@ -51,6 +51,109 @@ Use the same inline formatting as regular text:
 ##### The `:content_type` Option
 ```
 
+Code Examples
+-------------
+
+Wrap code examples with code fence syntax using backticks:
+
+````markdown
+```ruby
+puts "Hello World!"
+```
+````
+
+Always specify the language used in the example. Supported languages labels
+include: `ruby`, `html`, `irb`, `html+erb`, `bash` and `js`.
+
+````markdown
+```html+erb
```

Comment:
very very minor nit: should we also add `<% app/views/products/index.html.erb %>` in the file here.

Alternatively maybe this snippet can be something that doesn't look like it's referencing a specific file
````markdown
```html+erb
<p>Hello, <%= "World" %></p>
```
````